src/m365/aad/commands/app/app-get.spec.ts

```diff
@@ -145,21 +145,53 @@ describe(commands.APP_GET, () => {
   });
 
   it('fails validation if appId and objectId specified', async () => {
+    sinon.stub(cli, 'getSettingWithDefaultValue').callsFake((settingName, defaultValue) => {
```

Comment:
If we provide all required options in the `command.validate()` call below, then this shouldn't be necessary, right? And if the logic changes, and we add a required option that's missing, then the test will fail with a timeout. I wonder if that wouldn't be sufficient and let us avoid adding all this code to all validation tests.

If we don't want timeouts, then we should consider adding this stub in `before` and then cleaning it up in `after` to avoid repeating it. Since the logic to test required options is centralized and not a part of command, we won't need to unstub `cli.getSettingWithDefaultValue()` for specific tests, so it would save us some work to stub it once

Reply:
@Adam-it clarified offline that the current approach is necessary to support other test cases like disambiguation prompting. We'll this as-is